cmd/process-agent/subcommands/check/check.go

```diff
@@ -73,26 +74,35 @@ func nextGroupID() func() int32 {
 
 // Commands returns a slice of subcommands for the `check` command in the Process Agent
 func Commands(globalParams *command.GlobalParams) []*cobra.Command {
+	desc := "Run a specific check and print the results. Choose from: process, rtprocess, container, rtcontainer, connections, process_discovery, process_events"
+	return []*cobra.Command{MakeCommand(globalParams, "check", desc)}
+}
+
+func MakeCommand(globalParams *command.GlobalParams, name string, short string) *cobra.Command {
```

Comment:
```suggestion
func MakeCommand(globalParams *command.GlobalParams, name, short string) *cobra.Command {
```

Reply:
No longer relevant as the variable was changed